tests/test_controller.py

```diff
@@ -271,31 +418,29 @@ def assert_message(cls, message, identifier, status_code, description):
         eq_(str(status_code), cls.xml_value(message, 'simplified:status_code'))
         eq_(description, cls.xml_value(message, 'schema:description'))
 
+    @unauthenticated_request_context
```

Comment:
Using @unauthenticated_request_context lets us simplify the rest of the code in this test -- we don't have to create test request contexts because the entire test runs inside a single test request context.